.github/workflows/ci.yaml

```diff
@@ -23,7 +23,6 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v5
       with:
-        # renovate: datasource=golang-version depName=golang
         go-version: 1.21.6
```

Comment:
```suggestion
        go-version-file: go.mod
```

Reply:
Our docker image is already on 1.22?

Reply:
That's not good. We shouldn't build our release with a different version than defined in the go.mod

Reply:
Maybe we could add something to the Makefile to automatically inject the Golang version to the Docker build from the go.mod?